docs/user-guide/configure-zos-system.md

```diff
@@ -25,27 +33,48 @@ Be sure your z/OS system meets the following prerequisites:
-| Required if you have not run `ZWESECUR` and are configuring your z/OS environment manually. This step describes how to configure the cross memory server for SAF to guard against access by non-privileged clients. | [Configure the cross memory server for SAF](#configure-the-cross-memory-server-for-saf) |
-| Required for API Mediation Layer to map a client certificate to a z/OS identity. | [Configure main Zowe server to use client certificate identity mapping](#configure-main-zowe-server-to-use-client-certificate-identity-mapping) |
-| Required for API ML to map the association between a z/OS user ID and a distributed user identity. | [Configure main Zowe server to use distributed identity mapping](#configure-main-zowe-server-to-use-distributed-identity-mapping) |
-| To configure SAF Identity tokens on z/OS so that they can be used by Zowe components like zss or API Mediation Layer. | [Configure signed SAF Identity tokens IDT](#configure-signed-saf-identity-tokens-idt) |
-| Required for API Mediation Layer to issue SMF records. | [Configure the main Zowe server to issue SMF records](api-mediation/api-mediation-smf.md#configure-the-main-zowe-server-to-issue-smf-records) |
-| To use multi-factor authentication (MFA) | [Multi-Factor Authentication (MFA)](#multi-factor-authentication-mfa) |
-| To use Single Sign-On (SSO) | [Single Sign-On (SSO)](#single-sign-on-sso) |
-| To use OIDC Authentication with API Mediation Layer | [API Mediation Layer OIDC Authentication](#api-mediation-layer-oidc-authentication) |
+ Review the following table to determine which configuration steps are required based on your Zowe use case.
+
+| Purpose | Applicable Zowe Component(s) | Configuration step |
+| --- | --- | --- |
+| Set the names for the different z/OS UNIX address spaces for the Zowe runtime components. <br/>**Important:** This configuration step is required. | All components | [Configure address space job naming](#configure-address-space-job-naming) |
+| To use Zowe desktop. This step generates random numbers for zssServer that the Zowe desktop uses. | Application Framework | [Configure an ICSF cryptographic services environment](#configure-an-icsf-cryptographic-services-environment) |
+| To allow users to log on to the Zowe desktop through impersonation. |  | [Configure security environment switching](#configure-security-environment-switching) |
+| Required for TSS only. A TSS FACILITY needs to be defined and assigned to the `ZWESLSTC` started task. |  | [Configure multi-user address space for TSS only](#configure-multi-user-address-space-for-tss-only) |
+| Required if you have not run `ZWESECUR` and are manually creating the user ID and groups in your z/OS environment. |  | [Configure user IDs and groups for the Zowe started tasks](#configure-user-ids-and-groups-for-the-zowe-started-tasks) |
```

Comment:
All mention of "ZWESECUR" in this table should be rethought.

* ZWESECUR is not the recommended job to run anymore.
* It was also ever only one of now 4 ways to perform its actions (zwe, jcl, zosmf workflow, install wizard)
* Because the operations of ZWESECUR are instructed to be done within a prior doc page, care should be taken not to present this as something that needs to be done again.

You may wish to reword all "Required if you have not run `ZWESECUR`" entries to
"Required. Tasks are done within [zwe](./zwe-init-subcommand-overview#initializing-zowe-security-configurations-zwe-init-security), [workflow](./configure-zowe-zosmf-workflow), [install wizard](./install-zowe-server-install-wizard#configuring-the-zowe-server) or [jcl](./configuring-zowe-via-jcl)-based security setup.

Or

"Required. Tasks are done within [Zowe runtime configuration](./configure-zowe-runtime)"

Or to call them out in a separate table or list by saying "The following tasks are needed by Zowe, and are normally handled during Zowe runtime configuration. Read each link to learn more about the tasks performed by that configuration"